lib/v2/idolypride/news.js

```diff
@@ -0,0 +1,38 @@
+const { art } = require('@/utils/render');
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const path = require('path');
+
+module.exports = async (ctx) => {
+
+    const response = await got({
+        method: 'get',
+        url: 'https://idolypride.jp/news',
+    });
+    const data = response.data;
+    const $ = cheerio.load(data);
+    const list = $('div.lists__list li');
```

Comment:
Since this is a WordPress-based site, you are suggested to use WordPress API instead.

Official docs: https://developer.wordpress.org/rest-api/key-concepts/

All API endpoint: `/wp-json/wp/v2`
News endpoint: `/wp-json/wp/v2/news?per_page=100`